product-demos-ee/execution-environment.yml

```
@@ -10,9 +10,16 @@ dependencies:
   galaxy: requirements.yml
   python: requirements.txt
 
+additional_build_files:
+   - src: openshift-clients-4.12.0-202308011802.p0.g8669c6d.assembly.stream.el8.x86_64.rpm
+     dest: rpms
+
```

```suggestion
options:
  package_manager_path: /usr/bin/microdnf
```

Since you are using `ee-minimal-rhel8`, you would require to specify custom package manager.